deploy/lib/server_config.rb

```diff
@@ -2118,6 +2120,34 @@ def execute_query_7(query, properties = {})
     r
   end
 
+  def execute_query_8(query, properties = {})
+    # headers = {}
+
+    # params = {
+    #   :xquery => query,
+    #   :locale => LOCALE,
+    #   :tzoffset => "-18000"
+    # }
+
+    # r = go "#{@protocol}://#{@hostname}:#{@qconsole_port}/eval", "post", headers, params
+
+    headers = {
+      "Content-Type" => "application/x-www-form-urlencoded"
+    }
+
+    params = {
+      :xquery => query,
+      :locale => LOCALE,
+      :tzoffset => "-18000"
+    }
+
+    r = go "#{@protocol}://#{@hostname}:#{@qconsole_port}/v1/eval", "post", headers, params
+
+    raise ExitException.new(JSON.pretty_generate(JSON.parse(r.body))) if r.body.match(/\{"error"/)
```

Comment:
Did you verify that this line works as intended? I'm ignorant of the output of /v1/eval.